app/views/account/show.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class='container my-4'>
+  <div class='row'>
+    <div class='col-12'>
+      <h5>
+        <%= I18n.t('nav.account') %>
+      </h5>
+
+      <div class='card mt-4'>
+        <div class='card-body'>
+          <h4>Details</h4>
+
+          <div class="row">
+            <div class="col-md-2"><strong><%= t('activerecord.models.attributes.user.email') %>:</strong></div>
+            <div class="col-md-2"><%= current_user.email %></div>
```

Comment:
![Screenshot 2021-03-16 at 09 32 05](https://user-images.githubusercontent.com/40230316/111286890-9589e700-863a-11eb-8e33-f9d02ccc622f.png)

Something like col-md-8 would be better for the value to handle longer names/emails/user group names

Reply:
done, I'v set it to 10 to make row full width